ee/packages/media-calls/src/sip/Session.ts

```diff
@@ -0,0 +1,199 @@
+		});
+	}
+
+	public reportInternalCallUpdate(callId: string): void {
+		logger.debug({ msg: 'SipServerSession.reportInternalCallUpdate', callId });
+	}
+
+	public registerCall(call: BaseSipCall): void {
+		this.knownCalls.set(call.callId, call);
+	}
+
+	public configure(settings: IMediaCallServerSettings): void {
+		this.settings = settings;
+
+		if (!this.isEnabledOnSettings(settings)) {
+			return;
+		}
+
+		if (!this.wasEverEnabled) {
+			this.connectDrachtio();
+		}
+	}
+
+	public async createOutgoingCall(params: InternalCallParams): Promise<IMediaCall> {
+		return OutgoingSipCall.createCall(this, params);
+	}
+
+	public async createSipDialog(
+		sipExtension: string,
+		opts: Srf.CreateUACOptions,
+		progressCallbacks?: {
+			cbRequest?: (error: unknown, req: Srf.SrfRequest) => void;
+			cbProvisional?: (provisionalRes: Srf.SrfResponse) => void;
+		},
+	): Promise<Srf.Dialog> {
+		const uri = this.getExtensionUri(sipExtension);
+
+		return this.srf.createUAC(uri, opts, progressCallbacks);
+	}
+
+	public geContactUri(contact: MediaCallContact): string {
+		const sipExtension = contact.sipExtension || (contact.type === 'sip' && contact.id) || null;
+		const username = contact.username && `user-${contact.username}`;
+		const userId = contact.id && `user-${contact.id}`;
+
+		const sipUsername = sipExtension || username || userId;
+
+		return this.getExtensionUri(sipUsername || 'unknown');
+	}
+
+	public getExtensionUri(extension: string): string {
+		const { host, port } = this.settings.sip.sipServer;
+		if (!host) {
+			throw new Error('Sip Server Host is not configured');
+		}
+
+		const portStr = port ? `:${port}` : '';
+		return `sip:${extension}@${host}${portStr}`;
+	}
+
+	private isEnabledOnSettings(settings: IMediaCallServerSettings): boolean {
+		return Boolean(settings.enabled && settings.sip.enabled && settings.sip.drachtio.host && settings.sip.drachtio.secret);
+	}
+
+	private initializeDrachtio(): void {
+		logger.debug('Initializing Drachtio');
+		this.srf.on('connect', (err, hostport) => {
+			if (err) {
+				logger.error({ msg: 'Drachtio Connection Failed', err });
+				return;
+			}
+
+			logger.info({ msg: 'Connected to a drachtio server', hostport });
+		});
+
+		this.srf.on('error', (err: unknown, socket?: Socket) => this.onDrachtioError(err, socket));
+
+		this.srf.use((req, _res, next) => {
+			logger.info({ msg: 'Incoming message from Drachtio', method: req.method, source: req.source_address });
+			next();
+		});
+
+		this.srf.invite((req, res) => {
+			logger.info('Received a call on drachtio.');
+
+			void this.processInvite(req, res).catch((error) => {
+				logger.error({ msg: 'Error processing Drachtio Invite', error });
+			});
+		});
+	}
+
+	private connectDrachtio(): void {
+		if (this.wasEverEnabled) {
+			return;
+		}
+
+		const { host, port = 9022, secret } = this.settings.sip.drachtio;
+
+		logger.info({ msg: 'Connecting to drachtio', host, port });
```

Comment:
```suggestion
		logger.debug({ msg: 'Connecting to drachtio', host, port });
```

Reply:
This one I'll leave as `info` as it's slightly more useful than the debug ones